TODO.md

```diff
@@ -0,0 +1,40 @@
+This serves as a braindump for features that we want to scope for V1 (or seed a V2, V3, etc)
+
+# Project
+
+## Runtime
+
+- Node.js version? Do we have a minimum?
+  - @Nick: I would be fine with 14.x+ to jettison some legacy stuff but do not know if that would wall us off
+
+## Publication
+
+- Is there an NPM namespace / package we want to publish under? `@orkes/conductor-js` or `@conductor/client`
```

Comment:
I think we can use the same github handle? conductor-sdks/.. Assuming we can also replicate this for other languages such as Clojure, Python etc.